crates/nu-std/CONTRIBUTING.md

```diff
@@ -0,0 +1,191 @@
+# Contributing
+
+Welcome to the Nushell standard library and thank you for considering contributing!
+
+## Ideas for the standard library
+
+If you've got a great idea, or just want to contribute to open source
+by working on the Nushell standard library, 
+we invite you to talk to the team before you start coding. 
+You'll find we're friendly, passionate about Nushell and always open to new ideas!  
+
+You'll generally find the team members on
+[Discord standard-library channel](https://discord.com/channels/601130461678272522/1075541668922658868), 
+and can have preliminary discussions there to clarify the issues involved.
+
+You can open a [Github issue](https://github.com/nushell/nushell/issues/new/choose) 
+to have a more focused discussion of your idea.  
+
+Generally, we think the standard library should contain items that are 
+relevant to most/all Nushell users regardless of the application space they're working in. 
+If your idea isn't quite so broadly applicable, 
+consider publishing it in [nu_scripts](https://github.com/nushell/nu_scripts).
+
+Preliminary discussions should focus on the *user benefit* your idea would provide.  
+How many users will be affected by your idea, how would (a good implementation of) it 
+improve their Nushell experience?
+Given consensus on the user benefit, the team will be motivated to 
+help you create, deploy and maintain a solution long term.
+
+## Lifecycle of a change
+
+1. Verify the team thinks your idea is potentially relevant and useful, as above.
+1. If it's more than a simple bug fix, open a placeholder PR 
+as soon as you get started and mark it as work in progress (WIP).
+This will alert other contributors that you're working in this area and 
+advertise roughly what scope of changes you're thinking of.
+See [below](the_pr) for details.
```

Comment:
i think that should be `[below](#the-pr)` to jump to the anchor.

otherwise, when i read the document on your PR branch, it wants to bring me to the invalid https://github.com/bobhy/nushell/blob/stdlib_contributing/crates/nu-std/the_pr :scream:

Reply:
yes.